.github/workflows/pr-go.yaml

```diff
@@ -10,7 +10,7 @@ on:
 
 jobs:
   linter:
```

Comment:
Maybe we can separate the linter workflow from the test, wdyt? An example https://github.com/vimeda/mealplan-week-srv/blob/master/.github/workflows/linter.yaml

Reply:
Ok, updated